### PR TITLE
implementing LSTM model using tensorflow 2.x

### PR DESCRIPTION
**Pull request by antonionicampos**
Adding a notebook implementing original LSTM model code to tensorflow 2.x framework. 

- Code related to dataset held.
- LSTM model using tensorflow 2.x
- Using sklearn for the KFold cross-validation
- New code for results (confusion matrix, losses and accuracy history)

### CODE REVIEW

**Comment by gmxavier** [comment]
Thank you to contribute.
